qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -72,7 +78,25 @@ def __init__(
                 ansatz, the evaluated mean and the evaluated standard deviation.
             store_intermediate: Whether or not to store intermediate values of the optimization
                 steps. Per default False.
+            use_swap_strategies: A boolean on whether or not to use swap strategies when
+                transpiling. If this is False then the standard transpiler with the given
+                optimization level will run.
+            use_initial_mapping: A boolean flag that, if set to True (the default is False), runs
+                a heuristic algorithm to permute the Paulis in the cost operator to better fit the
+                coupling map and the swap strategy. This is only needed when the optimization
+                problem is sparse and when using swap strategies to transpile.
+            use_pulse_efficient: A boolean on whether or not to use a pulse-efficient transpilation.
+                If this flag is set to False by default. See https://arxiv.org/abs/2105.01063.
+            optimization_level: The transpiler optimization level to run if the swap strategies are
+                not used. This value defaults to 1 in the QAOA runtime.
+
+        Raises:
+            QiskitOptimizationError: if optimization_level is not None and use_swap_strategies
+                is True.
         """
+        if initial_point is None:
+            initial_point = np.random.rand(2 * reps)
```

Comment:
Do you need to set `initial_point` with random values in this method? I see `VQEClient` does not do it.
https://github.com/Qiskit/qiskit-optimization/blob/2a46fcd6878cd280e1140b83b08c9377ad377893/qiskit_optimization/runtime/vqe_client.py#L57-L58
https://github.com/Qiskit/qiskit-optimization/blob/2a46fcd6878cd280e1140b83b08c9377ad377893/qiskit_optimization/runtime/vqe_client.py#L81
If you need to set random values, you need an interface to set the random seed (`np.random.default_rng(seed)`) or it's good to use `qiskit.utils.algorithms_globals.random` instead of `np.random`.

Reply:
Fixed here: df972c8

Reply:
Could you explain why you reverted the change fd01687?

Reply:
`VQEClient` does not set random values in the constructor even if `initial_point` is None.
Why do you need to set random values in `QAOAClient`?
https://github.com/Qiskit/qiskit-optimization/blob/2a46fcd6878cd280e1140b83b08c9377ad377893/qiskit_optimization/runtime/vqe_client.py#L81